charts/rancher-backup/values.yaml

```diff
@@ -90,3 +90,19 @@ monitoring:
     additionalLabels: {}
     metricRelabelings: []
     relabelings: []
+  
+  alerts:
+    enabled: false
+
+    ## Define custom alerting rules here.
+    ## The "BackupFailed" alert is included by default when .Values.monitoring.alerts.enabled is set to true and rancher-monitoring is installed.
+    alertingRules:
+    - alert: BackupFailed
+      expr: rancher_backup{status!~"Completed|"} > 0
+      for: 1m
+      keep_firing_for: 10m
+      labels:
+          severity: critical
```

Comment:
AlertManager uses label matchers to route to specific integrations, so users will want to be able to pass in something along the lines of  `.alerts.default.labels` to add labels in a git-ops friendly way to the labels on the base alerts we provide